### ISSUE TITLE
Mid Install Reboot Catalina

### ISSUE THREAD

**Issue by gotjoshua** [SchmockLord/Hackintosh-Intel-i9-10900k-AsRock-Z490-Phantom-ITX-TB3#39]
So, i am pretty sure I have the Bios Settings correct.
and  
I managed to enable [OpenCore debugging](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/debug.html#config-changes).

I get to the selection screen for install or clear nvram, the apple screen comes up and the progress bar gets barely past halfway and then reboots.
I can't figure out how to get any debug info for whats going on behind that progress bar and what goes wrong at the end.
I tried CMD + L and CMD + V but nothing changes, it just reboots again and again...

any help?

config and boot log attached:
[config-082947.plist.txt](https://github.com/SchmockLord/Hackintosh-Intel-i9-10900k-AsRock-Z490-Phantom-ITX-TB3/files/5760266/config-082947.plist.txt)
[opencore-2021-01-02-082947.txt](https://github.com/SchmockLord/Hackintosh-Intel-i9-10900k-AsRock-Z490-Phantom-ITX-TB3/files/5760267/opencore-2021-01-02-082947.txt)

**Comment by PFalko** [comment]
What happens after the reboot? Mine took 3 reboots till the install was finished.

**Comment by gotjoshua** [comment]
Thanks for the reply!
well the bar never finishes, and the logs indicate an error at the end.  
   EDIT: and I also get the "your computer restarted because of a problem screen after reboot, before the apple and the loading bar come up.
the reboots always seem to start the install over again, i think i let it run for more than 5 times.
I will make a fresh install media and try another usb stick as well...
it WILL work.

**Comment by gotjoshua** [comment]
Small update:
I tried downloading the Catalina installer again and it was a slightly newer version: 10.15.03 vs 10.15.02
Also tried Big Sur 11.1 (fails even faster)
I also tried both OpenCore v 0.6.5 and 0.6.4 DEBUG

It seems to keep failing just after "SmcReadValue"...
Does anyone know a way to get more detailed panic info?

```
86:449 00:423 OC: Prelinked status - Success
89:298 02:849 AAPL: [EB|#SB:HW] j185fap\0
89:307 00:008 AAPL: [EB|#SB:MF] (System\\Library\\PrelinkedKernels\\immutablekernel.j185fap.im4m)
89:316 00:009 AAPL: [EB|`LD:LF] FIO: 0, DIR: 1, P: System\\Library\\PrelinkedKernels\\immutablekernel.j185fap.im4m, DP: 0
89:365 00:048 OCI4: Verifying digest 48 (EEA7F868) override 1 48 (EEA7F868)
92:188 02:823 OCI4: Matching override DB105189 with DB105189 - success
92:197 00:009 AAPL: [EB|#SB:MKRN]
92:232 00:034 AAPL: [EB|`LD:LKFS] } Ok(0)
92:241 00:009 AAPL: [EB|`LD:LKC] } Ok(0)
92:252 00:011 AAPL: [EB|#FWFM] 0xFFDFFF7F
92:261 00:009 AAPL: [EB|#FWFT] 0xFD8FF576
92:273 00:011 AAPL: [EB|#CSR:IN] 0x000003E7
92:283 00:009 AAPL: [EB|#CSR:OUT] 0x000003E7
92:292 00:009 AAPL: [EB|`BST:FBS] {
92:302 00:009 AAPL: [EB|`BST:FBS] BA.ADSZ: 0
92:311 00:008 AAPL: [EB|`BST:FBS] BA.KSDSZ: 0
92:321 00:010 OCSMC: SmcReadValue Key 4D535463 Size 1
92:330 00:009 OCSMC: SmcReadValue Key 4D534163 Size 2
92:339 00:008 AAPL: [EB|`B:WFDW] Err(0xE), 0 @ LocHB 71B4903C-14EC-42C4-BDC6-CE1449930E49
92:349 00:009 AAPL: [EB|#LOG:DT] 2021-01-04T19:49:11
92:359 00:010 AAPL: [EB|#LOG:EXITBS:START] 2021-01-04T19:49:11
```

I worked through the [OC troubleshooting for that EXITBS line](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/extended/kernel-issues.html#stuck-on-eb-log-exitbs-start) and tried pretty much every permutation of the options that they declare as relevant.

Any suggestions or help would be very appreciated!

**Comment by gotjoshua** [comment]
Success! I managed to get past the kernel panic (which was related to the on-board GPU).

Based on the [Desktop Comet Lake Guide](https://dortania.github.io/OpenCore-Install-Guide/config.plist/comet-lake.html#deviceproperties) I had to adapt the UHD 630 section:
```
<key>DeviceProperties</key>
	<dict>
		<key>Add</key>
		<dict>
			<key>PciRoot(0x0)/Pci(0x2,0x0)</key>
			<dict>
				<key>AAPL,ig-platform-id</key>
				<data>07009B3E</data>
				<key>framebuffer-patch-enable</key>
				<data>01000000</data>
				<key>framebuffer-stolenmem</key>
				<data>00003001</data>
				<key>model</key>
				<string>Intel UHD Graphics 630</string>
			</dict>
```

EDIT: I got quite a bit further than this and documented the process [in a gitlab issue on my fork](https://github.com/gotjoshua/Hackintosh-Intel-i9-10900t-AsRock-Z490-Phantom-ITX-TB3-iGPU/issues/1).
The above properties will boot but are far from optimal. Stolenmem was not needed in the end, and I used Base64 encoding as it seemed to get the most consistent results.
